vaadin-grid-flow-parent/vaadin-grid-flow/src/main/resources/META-INF/resources/frontend/gridConnector.js

```diff
@@ -463,6 +463,9 @@ import { isFocusable } from '@vaadin/grid/src/vaadin-grid-active-item-mixin.js';
             }
           });
         }
+        if (selectionMode === validSelectionModes[1]) { // NONE
+          row.removeAttribute('aria-selected');
```

Comment:
Seems that also the cell elements `<td>` have the attribute.

Reply:
I now added a check in the NONE test to check for the cells and made the appropriate change in the connector.